core-processor/src/handler.rs

```diff
@@ -35,7 +35,11 @@ pub fn handle_journal(
                 source,
                 outcome,
             } => handler.message_dispatched(message_id, source, outcome),
-            JournalNote::GasBurned { message_id, amount } => handler.gas_burned(message_id, amount),
+            JournalNote::GasBurned {
+                message_id,
+                amount,
+                is_panic: _,
+            } => handler.gas_burned(message_id, amount),
```

Comment:
You may insert it right in handler to prevent some future incompatibilities, but it's not a big deal

Reply:
Let's do it when we need to